ARIBA run with this command:
/software/pathogen/external/apps/usr/local/Python-3.6.0/bin/ariba prepareref prepareref -f /lustre/scratch118/infgen/pathdev/le8/serotype/Ariba_Test/test_SeroBA_11A_37/temp_aribaro82cek3/temp_fasta_genes.fasta -m /lustre/scratch118/infgen/pathdev/le8/serotype/Ariba_Test/test_SeroBA_11A_37/temp_aribaro82cek3/temp_meta_genes.tsv --cdhit_clusters /lustre/scratch118/infgen/pathdev/le8/serotype/Ariba_Test/test_SeroBA_11A_37/temp_aribaro82cek3/cdhit_cluster_genes database/ariba_db/10A/genes
from this directory: /lustre/scratch118/infgen/pathdev/le8/serotype/Ariba_Test/test_SeroBA_11A_37

ARIBA version: 2.9.3

External dependencies:
bowtie2	2.3.0	/software/pathogen/external/apps/usr/bin/bowtie2
cdhit	4.6	/software/pathogen/external/apps/usr/bin/cd-hit-est
nucmer	3.1	/software/pathogen/external/apps/usr/bin/nucmer

External dependencies OK: True

Python version:
3.6.0 (default, Jan 18 2017, 11:39:44) 
[GCC 4.6.3]

Python packages:
ariba	2.9.3	/software/pathogen/external/apps/usr/local/Python-3.6.0/lib/python3.6/site-packages/ariba/__init__.py
bs4	4.5.3	/software/pathogen/external/apps/usr/local/Python-3.6.0/lib/python3.6/site-packages/bs4/__init__.py
dendropy	4.2.0	/software/pathogen/external/apps/usr/local/Python-3.6.0/lib/python3.6/site-packages/dendropy/__init__.py
pyfastaq	3.15.0	/software/pathogen/external/apps/usr/local/Python-3.6.0/lib/python3.6/site-packages/pyfastaq/__init__.py
pymummer	0.10.2	/software/pathogen/external/apps/usr/local/Python-3.6.0/lib/python3.6/site-packages/pymummer/__init__.py
pysam	0.11.1	/software/pathogen/external/apps/usr/local/Python-3.6.0/lib/python3.6/site-packages/pysam/__init__.py

Python packages OK: True

Everything looks OK: True
